examples/workflows/script-annotations-outputs.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: test-output-annotations-
+spec:
+  entrypoint: my_steps
+  templates:
+  - name: my_steps
+    steps:
+    - - arguments:
+          parameters:
+          - name: a_number
+            value: '3'
+        name: script-param-artifact-in-function-signature-and-return-type
+        template: script-param-artifact-in-function-signature-and-return-type
+  - inputs:
+      parameters:
+      - name: a_number
+    name: script-param-artifact-in-function-signature-and-return-type
+    outputs:
+      artifacts:
+      - name: successor2
+      - name: successor4
+      parameters:
+      - name: successor
+      - name: successor3
+    script:
+      args:
+      - -m
+      - hera.workflows.runner
+      - -e
+      - examples.workflows.script_annotations_outputs:script_param_artifact_in_function_signature_and_return_type
+      command:
+      - python
+      env:
+      - name: hera__script_annotations
+        value: ''
```

Comment:
Maybe for the yaml readability we should put some meaningful value here? Like maybe `true`

Reply:
The problem is there isn't a meaningful value here (because we're going by existence of the envvar only), and adding something like `true` suggests `false` is a valid alternative but it's not.